### PR TITLE
Deploy QtLocation geoservices plugins

### PR DESCRIPTION
**Pull request by ntadej**
Deploy `QtLocation` geoservices plugins which are currently missing.

### CODE REVIEW

**Review by TheAssassin — APPROVED** [review]
Thanks! Looks good at a first glance. Let's see what the CI has to say.

**Comment by ntadej** [comment]
It seems there are some issues with the test inputs.

**Comment by TheAssassin** [comment]
GitHub actions doing GitHub actions things...

**Comment by ntadej** [comment]
I rebased now if the CI will pass. Let me test and confirm using the CI artifacts before merging.

File: src/deployers/CMakeLists.txt [hunk 1]
```diff
@@ -11,6 +11,7 @@ set(CLASSES
     SvgPluginsDeployer
     SqlPluginsDeployer
     PositioningPluginsDeployer
+    GeoServicesPluginsDeployer
```
[the remark this answers is not in the document]
Comment (ntadej) [reply src/deployers/CMakeLists.txt]: Done.

**Review by TheAssassin — APPROVED** [review]

**Comment by TheAssassin** [comment]
Thanks!

**Pull request merged**